packages/cdc-react/src/tests/mediaQueryUtils.ts

```diff
@@ -0,0 +1,17 @@
+const breakpoints = {
+  mobile: 0,
+  "mobile-lg": 480,
+  tablet: 640,
+  "tablet-lg": 880,
+  desktop: 1024,
+  "desktop-lg": 1200,
+  widescreen: 1400,
+};
+export type ScreenSize = keyof typeof breakpoints;
+
+const resizeView = (size: ScreenSize) => {
+  window.innerWidth = breakpoints[size];
+  window.dispatchEvent(new Event("resize"));
+};
+
+export { resizeView };
```

Comment:
Let's have consistent exporting of objects.  So either export each object individually, like you are doing on line 10.  Or export all in a single export statement like you are doing on line 17.